Automattic/jetpack
# Admin Page: add toggle to activate new Google Fonts feature

Follow-up to #23045

#### Changes proposed in this Pull Request:

This adds:

- a new check to the initial state passed by PHP, to indicate whether a site can use the webfonts feature.
- a new interface under Jetpack > Settings > Writing.

#### Jetpack product discussion

* paYE8P-1qo-p2

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start with a site where you have activated the Twenty Twenty Two theme.
* Install a development version of the Gutenberg theme (with latest `trunk`)
* Go to Jetpack > Settings > Writing and scroll down to Theme enhancements
    * You'll want to test other variations as well, when using another, non-block theme, or when not using the Gutenberg; the interface should not appear then.
* Ensure you can turn the feature on and off.
* Ensure the links work (the support doc is still a draft, so you will get a 404 on jetpack.com).
